src/common/ceph_context.h

```diff
@@ -258,7 +258,7 @@ class CephContext {
   std::string get_set_gid_string() const {
     return _set_gid_string;
   }
-
+  void leak_some_memory();
```

Comment:
so there is no need to have this declaration in header file, right? if that's the case, we can define it as a `static` function in .cc without this declaration.